presto-main/src/main/java/com/facebook/presto/spiller/NodeSpillConfig.java

```diff
@@ -21,6 +21,7 @@
 public class NodeSpillConfig
 {
     private DataSize maxSpillPerNode = new DataSize(100, DataSize.Unit.GIGABYTE);
+    private DataSize maxRevocableMemoryPerNode = new DataSize(8, DataSize.Unit.GIGABYTE);
```

Comment:
Default value is up for discussion, I will probably change this depending on how testing goes.